testing/sdk_utils.py

```diff
@@ -37,11 +37,20 @@ def get_zk_path(service_name):
     return "test__integration__" + service_name
 
 
+def is_test_failure(request):
```

Comment:
This feels kinda awkward here, but I get why you need it here. Could you please add a doc-string explaining that this is for ingesting pytest plugin requests, not the beloved requests lib used throughout this repo